crates/swc/src/config/mod.rs

```diff
@@ -598,6 +599,7 @@ impl Options {
                 }),
                 syntax.decorators()
             ),
+            paren_remover(Some(Box::leak(Box::new(comments)) as _)),
```

Comment:
I think this is wrong.
Do you have any idea about how to pass the `comments `? @kdy1

Reply:
You can simply clone it. https://rustdoc.swc.rs/swc_common/comments/trait.Comments.html#implementation-notes

Reply:
do you mean `paren_remover(comments.clone())`?
I got this error.

```
error[E0308]: mismatched types
   --> crates/swc/src/config/mod.rs:602:27
    |
602 |             paren_remover(comments.clone()),
    |             ------------- ^^^^^^^^^^^^^^^^ expected trait object `dyn Comments`, found struct `SingleThreadedComments`
    |             |
    |             arguments to this function are incorrect
    |
    = note: expected enum `std::option::Option<&dyn Comments>`
               found enum `std::option::Option<&SingleThreadedComments>`
```

Reply:
I think casting in `Option.map` may work